### PR TITLE
[+] Authentication - Users can connect to their project using Google Single Sign-On

### PR DESCRIPTION
**Pull request by ArnaudValensi**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [ ] Create automatic tests
- [x] No automatic tests failures
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by ArnaudValensi** [comment]
[(21) SSO / OAuth support](https://trello.com/c/fuuv6C98/2046-21-sso-oauth-support)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Authentication - Users can connect to their project using Google Single Sign-On.
 
 ## RELEASE 2.0.4 - 2018-01-08
 ### Changed
 - Performance - Set the CORS Max-Age to 1 day to be consistent with the other lianas behaviour.
+- Dependencies - Upgrade the json gem to work with Ruby versions 2.4.X on OSX.
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: Should be in the "Unreleased" section

File: app/services/forest_liana/google_authorized_user_getter.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+module ForestLiana
+  class GoogleAuthorizedUserGetter < UsersGetter
+    def initialize(rendering_id, forest_jwt)
+      @forest_jwt = forest_jwt
```
Comment (arnaudbesnier) [inline app/services/forest_liana/google_authorized_user_getter.rb]: I prefer `forest-token` it is simpler to understand than `jwt`

File: app/services/forest_liana/users_getter.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    def perform
+      http = Net::HTTP.new(@uri.host, @uri.port)
+      http.use_ssl = true if forest_url.start_with?('https')
+
+      begin
+        http.start do |client|
+          request = Net::HTTP::Get.new(@uri.path)
+          request['Content-Type'] = 'application/json'
+          request['forest-secret-key'] = ForestLiana.env_secret
+          request['forest-jwt'] = @forest_token if @forest_token
```
Comment (arnaudbesnier) [inline app/services/forest_liana/users_getter.rb]: sorry to insist but If you rename it, it should be everywhere, `forest-jwt` remains here.

Comment (ArnaudValensi) [reply app/services/forest_liana/users_getter.rb]: yep, just saw it

**Pull request merged**